src/config/wmodules-aws.c

```diff
@@ -398,12 +372,15 @@ int wm_aws_read(const OS_XML *xml, xml_node **nodes, wmodule *module)
 
             OS_ClearNode(children);
 
-        } else {
```

Comment:
Done, I need to add a litttle code here since now we need to verify it is not a scheduling tag before declaring it invalid.
I have also added a test in each module to test that an `OS_INVALID` is returned when there is an unexpected tag in the module configiration